pkg/lexer/lexer.go

```diff
@@ -113,7 +113,8 @@ func isIdentifierChar(ch byte) bool {
 }
 
 func isLiteralChar(ch byte) bool {
-	return isLetter(ch) || isNumber(ch)
+	// ToDo (suggestion): return ch != '"'
+	return isLetter(ch) || isNumber(ch) || ch == '.'
```

Comment:
`isLiteralChar` used only in `readLiteral` method, that's used only in case it's needed to read string... and in this way dot is only the part of string.

Reply:
i'm going to open an issue to deal with readLiteral() - the logic is "incorrect".

what you have now is fine in the interim

Reply:
this will be dealt with in a separate pr:
https://github.com/infobloxopen/seal/issues/85